components/purchase_modal/purchase_modal.tsx

```diff
@@ -527,7 +531,7 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
                 </div>
                 <div className='RHS'>
                     <div className='price-container'>
-                        {(this.props.products && Object.keys(this.props.products).length > 1) &&
+                        {(this.props.products && Object.keys(this.props.products).length > 1 && this.state.currentProduct?.sku !== 'cloud-professional') &&
```

Comment:
Why do we need to hardcode the condition here to check for product name? If there is more than 1 product then it should have the compare plans. In the case of current sku is `Starter` there should be more than one product so it should show the compare plans and in the case of `Professional` it should only have 1 product and not show this

Reply:
ok